diff --git a/node_modules/react-native-quick-base64/android/build.gradle b/node_modules/react-native-quick-base64/android/build.gradle
index 6a104e7..6346e56 100644
--- a/node_modules/react-native-quick-base64/android/build.gradle
+++ b/node_modules/react-native-quick-base64/android/build.gradle
@@ -42,6 +42,10 @@ android {
 
   compileSdkVersion getExtOrIntegerDefault("compileSdkVersion")
    
+  if (rootProject.hasProperty("ndkVersion")) {
+    ndkVersion rootProject.ext.ndkVersion
+  }
+
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault("minSdkVersion")
     targetSdkVersion getExtOrIntegerDefault("targetSdkVersion")
