diff --git a/node_modules/react-native-worklets-core/android/build.gradle b/node_modules/react-native-worklets-core/android/build.gradle
index b6af1c3..9123243 100644
--- a/node_modules/react-native-worklets-core/android/build.gradle
+++ b/node_modules/react-native-worklets-core/android/build.gradle
@@ -72,6 +72,10 @@ task deleteCmakeCache() {
 android {
   compileSdkVersion getExtOrIntegerDefault("compileSdkVersion")
   
+  if (rootProject.hasProperty("ndkVersion")) {
+    ndkVersion rootProject.ext.ndkVersion
+  }
+
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault("minSdkVersion")
     targetSdkVersion getExtOrIntegerDefault("targetSdkVersion")
